index.html

```diff
@@ -1,29 +1,136 @@
 <!DOCTYPE html>
 <html lang="en-ca">
-  <head>
-    <!-- Set the page title -->
-    <title>Assignment 02</title>
+<head> 
+   <!--This section creates the title of the HTML document and links it the CSS file -->
+    <meta charset = "utf-8">
+    <title> Pepper Pete's Website</title>
+    <link rel="stylesheet" href="style.css">
+</head>
 
-    <!-- Set the charset we're using to UTF-8 -->
-    <meta charset="UTF-8" />
+<body>  <!--This section outlines the header logo, title and header list -->
+   <header> <div class="head"> 
+      <img width="80" src="logo_v3.png" alt="header logo">
+      <h1 style="color:aliceblue; background-color: #E0643C; margin-right: auto"> PEPPER PETE'S</h1> 
+   
+   <!--these are the menu options -->
+   <ul class="nav-list"> 
+      <li> <a href="Home.html"> Home </a></li>
+      <li> <a href="Menu.html"> Our Menu </a></li>
+      <li> <a href="About_Us.html"> About Us </a></li>
+      <li> <a href="Location.html"> Locations </a></li>
+      <li> <a href="Events.html"> Events </a></li>
+      <li> <a href="Nutrition.html"> Nutrition </a></li>
+   </ul>
+</div> </header>
 
-    <!-- Set the screens default zoom / scale to 1 -->
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
+<!-- Displays the image and text for the hero section, in this case it's our new stir fry -->
+<div class = stir>  
+   <img width = "600" src="stir_fry.jpg" alt="stir fry image">
+   <h2> Try Our New <br> Frightening <br> Ghost Pepper <br> Chicken Stir Fry</h2>
+</div>
+
+<!-- Displays the text for the 'Where Heat is Your Treat' quote -->
+<div class = treat> 
+   <h2> "Where Heat Is Your Treat" <br>-Guy Fieri </h2>
+</div>
     
-    <!-- Specifies the document mode for Internet Explorer -->
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+<!--This class the 'Our Menu' section, include the titles, image, and button -->
+ <div class = menu> 
+   <h2>Our Menu</h2>
+   <img class = "our_menu" width = "500" src="blurred_menu.png" alt="image of woman holding menu">
+   <h3> Learn More About The Food <br> We Serve:</h3>
+   <button type="button"> View Full Menu </button>
+   <div class = food> <img width = 600 src="circles_all.png" alt="images of food in circles"> </div> 
+</div>
```

Comment:
Please use a link `<a>` here instead of a button. 

`<buttons>` should generally be used for web application interactions or interacting with form elements. 

Note: I'm making the assumption that "View full menu" takes the user to a different page where they can view the full menu